spec/xcpretty/parser_spec.rb

```diff
@@ -444,6 +444,11 @@ module XCPretty
           @parser.parse(line)
         end
       end
+
+      it "parses ld warnings" do
+        @formatter.should receive(:format_ld_warning).with("ld: embedded dylibs/frameworks only run on iOS 8 or later")
```

Comment:
Line is too long. [119/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.